README.md

```diff
@@ -179,10 +186,9 @@ Commonly enabled services that you may wish to disable:
 
 The `examples` directory contains the `example.krun` experiment. This contains
 two benchmark programs (*nbody* and *dummy*), both of which are run on *C* and
-*CPython* (both of which we'll consider to be "VMs"). Each benchmark is run for
-5 *in-process iterations* (where the benchmark is repeated 5 times within a for
-loop within a single process) across 2 *process executions* (where the entire VM
-is restarted).
+*CPython*. Each benchmark is run for 5 *in-process iterations* (where the
```

Comment:
You meant PyPy?

https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/examples/example.krun#L38-L42

Reply:
Oops...